avm/res/automation/automation-account/tests/e2e/max/main.test.bicep

```diff
@@ -265,6 +265,17 @@ module testDeployment '../../../main.bicep' = [
           value: '\'TestEncryptedValue\''
         }
       ]
+      webhooks: [
+        {
+          name: 'TestWebhook'
+          runbookName: 'TestRunbook'
+          expiryTime: '2025-12-31T13:00'
```

Comment:
Question: will this become a problem in the future when we're past december? If the case, it might be better to use a dynamic expression here, using e.g. the DateFunction: https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/bicep-functions-date . Thoughts?